composer.json

```
@@ -22,23 +22,23 @@
         }
     ],
     "require": {
-        "php": "^7.1",
-        "symfony/config": "^3.4 || ^4.2",
-        "symfony/dependency-injection": "^3.4 || ^4.2",
-        "symfony/http-foundation": "^3.4 || ^4.2",
-        "symfony/http-kernel": "^3.4 || ^4.2",
-        "symfony/intl": "^3.4 || ^4.2",
-        "symfony/templating": "^3.4 || ^4.2",
+        "php": "^7.2",
+        "symfony/config": "^3.4 || ^4.2 || ^5.0",
+        "symfony/dependency-injection": "^3.4 || ^4.2 || ^5.0",
+        "symfony/http-foundation": "^3.4 || ^4.2 || ^5.0",
+        "symfony/http-kernel": "^3.4 || ^4.2 || ^5.0",
+        "symfony/intl": "^3.4 || ^4.2 || ^5.0",
+        "symfony/templating": "^3.4 || ^4.2 || ^5.0",
         "twig/twig": "^2.9"
     },
     "conflict": {
         "sonata-project/user-bundle": "<2.0 || >=5.0"
     },
     "require-dev": {
-        "matthiasnoback/symfony-dependency-injection-test": "^3.1",
+        "matthiasnoback/symfony-dependency-injection-test": "^3.1 || ^4.0",
```

```suggestion
        "matthiasnoback/symfony-dependency-injection-test": "^4.0",
```
Using PHPUnit 8 it should be 4.0